docs/static/monitoring/monitoring-internal.asciidoc

```diff
@@ -2,10 +2,10 @@
 [[monitoring-internal-collection]]
 === Collect {ls} monitoring data using internal collectors
 ++++
-<titleabbrev>Internal collection (deprecated)</titleabbrev>
+<titleabbrev>Internal collection</titleabbrev>
 ++++
 
-deprecated[7.7.0]
+IMPORTANT: This functionality will be deprecated in 7.7.0
```

Comment:
What do you think about "This functionality is scheduled for deprecation in `7.7.0`"?

Reply:
Works for me.